fvm/src/call_manager/default.rs

```diff
@@ -412,6 +416,198 @@ where
         Ok(())
     }
 
+    fn upgrade_actor<K: Kernel<CallManager = Self>>(
```

Comment:
We'll have to make sure to wrap this in a transaction. I.e., share the logic in `send`.

Which is annoying because I just removed the separate "transaction" abstraction... but oh well...